documentation/staging/content/managing-domains/domain-lifecycle/retry.md

```diff
@@ -0,0 +1,116 @@
+Reason:                ReplicasTooHigh
+Severity:              Warning
+Status:                True
+Type:                  Failed
+   ...
+```
+{{% /expand %}}
+
+### Retry behavior
+
+Domains that have failures with a severity of `Fatal` or `Warning` will not be retried. The domain status should contain a message indicating what action is needed to fix the failure condition. 
+
+Domains failures with a severity of `Severe` will be retried as follows:
+- The operator calculates the next retry time based on the timestamp of the previous failure, so the next retry always occurs at the time that is the last failure timestamp plus a predefined retry interval.
+  - The timestamp of the previous failure can be found in the `lastFailureTime` field in the domain status. 
+  - The retry interval is specified in the `failureRetryIntervalSeconds` field in the Domain spec. It has a default 
+    value of 120 seconds. A value of zero seconds means retry immediately after failure.
+- The operator stops retrying a domain resource when the time elapsed since the initial failure exceeds a predefined maximum retry time.
+    - The timestamp of the initial failure can be found in the `initialFailureTime` field the domain status.
+    - The retry interval is specified in the `failureRetryLimitMinutes` field in the Domain spec. It has a default value of 1440 minutes (24 hours).
+      A value of zero minutes will disable retries, which can be useful for accessing log files for debugging purposes.
+
+The following is an example of domain status showing a failure with pending retries. This Domain resource is configured to have a 
+`failureRetryLimitMinutes` of 10 minutes. Note that the next retry is 120 seconds after the `Last Failure Time`,
+and the retry until time is 10 minutes after the `Initial Failure Time`.
+```yaml
+Status:
+  ...
+  Initial Failure Time:      2022-10-11T23:16:21.851801Z
+  Last Failure Time:         2022-10-11T23:21:53.109997Z
+  Message:                   Failure on pod 'domain1-introspector-hlvwt' in namespace 'default': Back-off pulling image "oracle/weblogic:12214". Will retry next at 2022-10-11T23:23:53.109997240Z and approximately every 120 seconds afterward until 2022-10-11T23:26:21.851801Z if the failure is not resolved.
+```
+
+In this example, all retries failed to start the domain before the predefined retry time limit, and the domain status shows a `Fatal` failure with `Aborted` reason. 
+```yaml
+Status:
+  Clusters:
+  Conditions:
+    Last Transition Time:    2022-10-11T23:26:34.107662Z
+    Message:                 The operator failed after retrying for 10 minutes. This time limit may be specified in spec.failureRetryLimitMinutes. Please resolve the error and then update domain.spec.introspectVersion to force another retry.
+    Reason:                  Aborted
+    Severity:                Fatal
+    Status:                  True
+    ...
+```
```

Comment:
To manually initiate an immediate retry, or to restart retries that have reached their `domain.spec.failureIntervalRetrySections`, update a domain field that will cause immediate action by the operator. For example, change 'spec.introspectVersion' or 'spec.restartVersion' as appropriate, see [userguide/managing-domains/domain-lifecycle/startup #fields-that-cause-servers-to-be-restarted] and [userguide/managing-domains/domain-lifecycle/introspection/#initiating-introspection].